tests/cases/conformance/jsx/tsxLibraryManagedAttributes.tsx

```diff
@@ -0,0 +1,126 @@
+// @jsx: preserve
+// @strict: true
+
+type Defaultize<TProps, TDefaults> =
+    & {[K in Extract<keyof TProps, keyof TDefaults>]?: TProps[K]}
+    & {[K in Exclude<keyof TProps, keyof TDefaults>]: TProps[K]}
+    & Partial<TDefaults>;

```

Comment:
Why is the `Partial<TDefaults>` here intersected with the extracted mapped type from `TDefaults`? What kind of error does that produce if `typeof TProps[K]` is incompatible with `typeof TDefaults[K]`?